server/index.js

```diff
@@ -151,6 +152,35 @@ app.get('/api/projects/titles/:projectId', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.post('/api/projects/create', (req, res, next) => {
+  const asssignedTo = req.user.userId;
+
+  const { projectName } = req.body;
+  const userId = req.body.users[0].userId;
```

Comment:
I'm kinda confused. Who exactly is the assignee? I thought you didn't assign projects to users, just tasks?